tests/e2e/tutorials/rclone_large.js

```diff
@@ -0,0 +1,88 @@
+// node rclone_small.js [url] [--user user] [--pass password] [--demo]
+
+const utils = require('../utils/utils');
+const tutorialBase = require('./tutorialBase');
+
+const args = process.argv.slice(2);
+const {
+  url,
+  user,
+  pass,
+  newUser,
+  startTimeout,
+  basicauthUsername,
+  basicauthPassword,
+  enableDemoMode
+} = utils.parseCommandLineArguments(args)
+
+const templateName = "rclone -- large";
+
+async function runTutorial() {
+  const tutorial = new tutorialBase.TutorialBase(url, templateName, user, pass, newUser, basicauthUsername, basicauthPassword, enableDemoMode);
+  let studyId;
+  try {
+    await tutorial.start();
+    const studyData = await tutorial.openTemplate(1000);
+    studyId = studyData["data"]["uuid"];
+
+    const workbenchData = utils.extractWorkbenchData(studyData["data"]);
+    await tutorial.waitForServices(
+      workbenchData["studyId"],
+      [workbenchData["nodeIds"][1], workbenchData["nodeIds"][2]],
+      startTimeout,
+      false
+    );
```

Comment:
Here you are waiting for the 2nd and 3rd services... I guess you want to wait for all of them.

Reply:
indeed